src/language-graphql/printer-graphql.js

```diff
@@ -288,7 +288,7 @@ function genericPrint(path, options, print) {
               " implements ",
               join(
                 determineInterfaceSeparator(
-                  options.originalText.substr(
+                  options.originalText.slice(
                     options.locStart(n),
                     options.locEnd(n)
```

Comment:
This doesn't look right.

Reply:
Good catch, not equal. But I think old one is wrong, will think a case to test and confirm

Reply:
fixed in #7305